documentation/course/e2/README.md

```diff
@@ -2047,7 +2056,36 @@ TODO @jakubzehner: Dodać diagram pakietów, opis architektury i endpointy.
 
 ## Płatność
 
-TODO @piterek130: Dodać diagram pakietów, opis architektury i endpointy.
+TODO @piterek130: Dodać diagram pakietów, opis architektury.
+
+### API
+
+#### API publiczne
+
+| **Rola**                 | **Metoda** | **Endpoint**                   | **Wymagania**                | **Opis**                                    |
+| ------------------------ | ---------- | ------------------------------ | ---------------------------- | ------------------------------------------- |
+| `passenger`              | `GET`      | `/ext/v1/cards`                | `PAY/19`                     | Pobranie listy kart płatniczych.            |
+| `passenger`              | `POST`     | `/ext/v1/cards`                | `PAY/02`                     | Dodanie nowej karty płatniczej.             |
+| `passenger`              | `PUT`      | `/ext/v1/cards/:id`            | `PAY/20`                     | Zaktualizowanie danych karty płatniczej.    |
+| `passenger`              | `DELETE`   | `/ext/v1/cards/:id`            | `PAY/03`                     | Usunięcie karty płatniczej.                 |
+| `passenger`              | `POST`     | `/ext/v1/wallet/add-funds`     | `PAY/07`                     | Doładowanie portfela.                       |
+| `passenger`              | `POST`     | `/ext/v1/wallet/fines/:id/pay` | `PAY/14`, `PAY/15`, `PAY/16` | Realizacja płatności mandatu.                       |
```

Comment:
Why not `/ext/v1/fines/:id/pay`? I'd locate it under `GET /ext/v1/fines/:id` instead of here.